packages/plugin-css/test/color/want.css

```diff
@@ -47,35 +47,35 @@
 
 @supports (color: color(display-p3 1 1 1)) {
   :root {
-    --ds-color-blue: color(display-p3 0.129412 0.545098 1);
-    --ds-gradient: color(display-p3 0.129412 0.545098 1) 0%, color(display-p3 0.909804 0.352941 0.678431) 100%;
+    --ds-color-blue: color(display-p3 0.12941176470588237 0.5450980392156862 1);
+    --ds-gradient: color(display-p3 0.12941176470588237 0.5450980392156862 1) 0%, color(display-p3 0.9098039215686274 0.35294117647058826 0.6784313725490196) 100%;
```

Comment:
FWIW I’m glad I maintained my own color science library for a bit. If no other reason than to be sure the colors were accurate. With thousands of ways to do things in culori, it’s so easy to produce the wrong colors when upconverting to P3, and it would take a while to realize it.